components/card/card.js

```diff
@@ -1 +1,43 @@
-export function createCharacterCard() {}
+export const cardContainer = document.querySelector(
+  '[data-js="card-container"]'
+);
+
+export function createCharacterCard(characterData) {
+  const newCard = document.createElement("li");
+  newCard.classList.add("card");
+  cardContainer.append(newCard);
+
+  const src = "";
+  src.textContent = characterData.image;
+  const name = "";
+  name.textContent = characterData.name;
+  const status = "";
+  status.textContent = characterData.status;
+  const type = "";
+  type.textContent = characterData.type;
+  const occurrences = "";
+  occurrences.textContent = characterData.episode.length;
```

Comment:
This approach won't work given that the values 
```js
characterData.image
characterData.name
characterData.status
characterData.type
characterData.episode.length
```
return a string and the values you try to pass them into
namely
```js
const src = "";
const name = "";
const status = "";
const type = "";
const occurrences = "";
```

are strings
you can't do that with the 
`.textContent` property. That property doesn't exist in `string`s as far as I know that is only an HTMLElement property.